spec/indexers/cypripedium_indexer_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require 'rails_helper'
 
-RSpec.describe PublicationIndexer do
+RSpec.describe CypripediumIndexer do
   let(:indexer) { described_class.new(publication) }
```

Comment:
Would it be possible to have the same tests on the other models, or use the pattern for the shared examples in spec/support/shared_examples?